src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Transaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Transaction> transactions = new ArrayList<>();
```

Comment:
Storage should contain fruits and their amount, not transactions

Reply:
more convenient to use `Map` but if you want to use `List` create some class that will contain fruit name and current amount
and store objects of this class in the list

your solution will change greatly after you fix this comment